CREDITS.md

```diff
@@ -38,11 +38,13 @@ Contributors to the Firecracker repository:
 * Andrii Radyk <ander.ender@gmail.com>
 * Angus McInnes <angus@amcinnes.info>
 * Arun Gupta <arun.gupta@gmail.com>
+* Arunav Sanyal <Khalian@users.noreply.github.com>
 * Atsushi Ishibashi <atsushi.ishibashi@finatext.com>
 * Aussie Schnore <aussiev123@yahoo.com>
 * Babis Chalios <babis.chalios@gmail.com>
 * Begley Brothers Inc <begleybrothers@gmail.com>
 * Benjamin Fry <benjaminfry@me.com>
+* berciuliviu <lberciu@amazon.com>
```

Comment:
Should be formatted as `Liviu Berciu`. You could edit the `.mailmap` file

Reply:
Fixed.